tests/protractor/e2e/forms/submit-delivery-form.specs.js

```diff
@@ -28,6 +82,26 @@ describe('Submit Delivery Report', () => {
       return element(by.css('#report-form')).isPresent();
     }, 10000);
 
-    expect(helper.isTextDisplayed('Delivery'));
+    //select name
+    deliveryReport.selectPatientName('jack');
+    deliveryReport.nextPage();
+    helper.waitElementToBeVisisble(element(by.css('[value="healthy"]')));
+
+    //Delivery info
+    deliveryReport.selectLiveBirthButton();
+    deliveryReport.selectFacilityButton();
+    deliveryReport.enterDeliveryDate('');
+    deliveryReport.nextPage();
+    expect(deliveryReport.getNoteToCHW()).toBe(noteToCHW);
+    deliveryReport.nextPage();
+
+    //summary page
+    expect(deliveryReport.getOutcomeText()).toBe('Live Birth');
```

Comment:
I used a variable for 'note to chw' because it is a relatively long string, which also contains other variables (although hard-coded for now).